data/issuance-protocols/Coconut.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "../../schemas/issuance-protocol.json",
+  "Name": "Coconut"
```

Comment:
is there a link to a specification?

Reply:
Same as https://github.com/openwallet-foundation/digital-wallet-and-agent-overviews-sig/pull/138/files#diff-cdcf2335b6a30d3cd8947c5c43fcba291710b3c21b59dcba85bc2836edf658b5R5 but saw that not all the other issaunce-protocols defines them, I can surely add it